artemis-server/src/main/java/org/apache/activemq/artemis/core/server/files/FileStoreMonitor.java

```diff
@@ -73,8 +75,8 @@ public FileStoreMonitor addStore(File file) throws IOException {
             try {
                addStore(Files.getFileStore(file.toPath()));
             } catch (IOException e) {
-               logger.error("Error getting file store for " + file.getAbsolutePath(), e);
-               throw e;
+               logger.warn("Error getting file store for " + file.getAbsolutePath() + ". Trying dummy FileStore", e);
```

Comment:
This will mean now real filesystems where we need to trust the store has been successful will silently fail. E.g. if permission or actual io issue with disk

Reply:
You will need to differentiate file systems if youre doing this as majority will be using normal disk filestores and expecting error if unsuccessful

Reply:
Thanks for the valuable feedback. We recently upgraded from Wildfly 10 to 17. This patch was intended to fix  our product regressions which run on RAM disk.